CONTRIBUTING.md

```diff
@@ -85,7 +88,7 @@ If you would like to help and are able to contribute code, you are most welcome.
 
 There are many reasonable feature requests and ideas for new question types in the issue tracker which you could also engage yourself with. If you have own ideas how to improve this app and want to make sure that the Pull Request will be merged, it is strongly suggested to **open an issue first** to discuss the feature.
 
-If you need to find things, where help is appreciated
+If you need to find things, where help is appreciated [have a look at the issues tagged with "help wanted"](https://github.com/westnordost/StreetComplete/labels/help%20wanted).
```

Comment:
"things, where help is appreciated" -> "things where help is appreciated,"

Reply:
I see I really use too much commas… :laughing: